packages/components/src/quantity/src/components/osds-quantity/interfaces/attributes.ts

```diff
@@ -1,10 +1,3 @@
-interface OdsQuantityAttribute {
-  /**
-   * Quantity is disabled or not
-   */
-  disabled?: boolean
-}
+import type { OdsCommonFieldAttribute } from '@ovhcloud/ods-common-core';
 
-export {
-  OdsQuantityAttribute,
-};
+export type { OdsCommonFieldAttribute as OdsQuantityAttribute };
```

Comment:
You need to omit some key that is not used in the quantity
`clearable
forbiddenValues
label
loading
placeholder
prefixValue
readonly
required
step
tabindex`